generate.go

```diff
@@ -0,0 +1,8 @@
+package regen_ledger
+
+//go:generate mockgen -source=x/data/expected_keepers.go -package mocks -destination x/data/mocks/expected_keepers.go
+//go:generate mockgen -source=x/ecocredit/server/core/keeper.go -package mocks -destination x/ecocredit/server/core/mocks/keeper.go
+//go:generate mockgen -source=x/ecocredit/expected_keepers.go -package mocks -destination x/ecocredit/mocks/expected_keepers.go
+//go:generate mockgen -source=x/ecocredit/expected_keepers.go -package mocks -destination x/ecocredit/mocks/expected_keepers.go
+//go:generate mockgen -source=x/ecocredit/server/basket/keeper.go -package mocks -destination x/ecocredit/server/basket/mocks/keeper.go
+//go:generate mockgen -source=x/ecocredit/expected_keepers.go -package mocks -destination x/ecocredit/mocks/expected_keepers.go
```

Comment:
normally we put the directive next to the related types. So the line above should be placed in x/ecocredit/expected_keepers.go file.

Reply:
Can do. I found this as an example but looks like best practice is on a per file basis.

Reply:
I see why this was an example. You have to run `go generate ./...` from within the directory and this was being used as a convenient way to generate all mocks with one command.